action.yml

```diff
@@ -4,9 +4,14 @@ branding:
   icon: code
   color: blue
 inputs:
+  balena_api_key:
+    description: Your balenaCloud API key.
+    required: false
+    default: ""
   balena_token:
-    description: Your balenaCloud API token
-    required: true
+    description: Deprecated; Use balena_api_key instead.
```

Comment:
Just realized, can you also add a warning if it is used like in https://github.com/product-os/flowzone/blob/v0.35.13/.github/workflows/flowzone.yml#L906 eg
```bash
echo "::warning::balena_token is deprecated, please use balena_api_key instead"
```

Reply:
Oh cool! TIL, will do

Reply:
Done https://github.com/balena-io/deploy-to-balena-action/pull/214/files#diff-1243c5424efaaa19bd8e813c5e6f6da46316e63761421b3e5f5c8ced9a36e6b6R71-R72, this should work if I'm reading the docs correctly.